front50-web/src/main/groovy/com/netflix/spinnaker/front50/controllers/PermissionsController.groovy

```diff
@@ -107,6 +112,36 @@ public class PermissionsController {
     syncUsers(null, oldPermission)
   }
 
+  @ApiOperation(value = "", notes = "Get all permissions for a given resource type")
+  @RequestMapping(method = RequestMethod.GET, value = "/groups")
+  Set<GroupPermission> getAllGroupPermissions(@RequestParam(required = false) String resourceType) {
+    if (resourceType != null) {
+      ResourceType parsedResourceType = ResourceType.parse(resourceType);
+      return groupPermissionDAO.findAllByResourceType(parsedResourceType);
+    } else {
+      return groupPermissionDAO.all();
+    }
+  }
+
+  @ApiOperation(value = "", notes = "Create a new group permission")
+  @RequestMapping(method = RequestMethod.POST, value = "/groups")
+  GroupPermission createGroupPermission(@RequestBody GroupPermission groupPermission) {
+    return groupPermissionDAO.create(groupPermission.getId(), groupPermission);
+  }
+
+  @ApiOperation(value = "", notes = "Update an existing group permission")
+  @RequestMapping(method = RequestMethod.PUT, value = "/groups/{groupId}")
+  GroupPermission updateGroupPermission(@PathVariable String groupId, @RequestBody GroupPermission groupPermission) {
+    groupPermissionDAO.update(groupId, groupPermission);
+    return groupPermission;
```

Comment:
we should look at what happens when we save a ServiceAccount - I think there we do a callout to fiat to re-sync roles, we probably need to do the same here

Reply:
(across create, update, and delete)